src/Component/Card.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import countries from '../data/countries.json';
+import "../App.css";
+
+export const Cards: React.FC = () => {
+  return (
+    //  Cards component
+    <div className="cards-container">
+      {countries.map((country, index) => (
+        <div className="card" key={index}>
+          <img src={country.flags.png} alt={`Flag of ${country.name.common}`} />
```

Comment:
Image aspect ratio is quite off, -3
<img width="352" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a2/assets/62576642/40972a49-2148-4a45-a60c-6d0ae2663ea3">